pkg/clusters/addons/kong/kong.go

```diff
@@ -29,6 +29,18 @@ const (
 	AddonName clusters.AddonName = "kong"
 
 	httpPort = 80
+
+	// DefaultEnterpriseImageRepo default kong enterprise image
+	DefaultEnterpriseImageRepo = "kong/kong-gateway"
+
+	// DefaultEnterpriseImageTag latest kong enterprise image tag
+	DefaultEnterpriseImageTag = "2.5.0.0-alpine"
+
+	// KongLicenseSecretName is the kong license data secret name
+	KongLicenseSecretName = "kong-enterprise-license"
+
+	// EnterpriseLicense is the kong enterprise test license
+	EnterpriseLicense = "KONG_ENTERPRISE_LICENSE"
```

Comment:
We need to start shifting environment handling into the CLI and have options for the library simply be parameters, we should remove this env var here and just take an option for the license in the `Builder`.

```suggestion
```

Reply:
Builder as following ?
type Builder struct {
	namespace  string
	enterprise-license        string  // the enterprise license secret name 
}

func (b *Builder) WithEnterpriseLicense(license string) *Builder {
   b.enterprise-license = license
}

Reply:
e02d137c22f069e1af34af42d9e3367a1e7e20b3